project/cfg_utils.py

```diff
@@ -0,0 +1,15 @@
+"""Utilities for working with CFG"""
+from pyformlang.cfg import CFG
+
+__all__ = ["cfg_to_cnf", "cfg_from_file"]
+
+
+def cfg_to_cnf(cfg: CFG) -> CFG:
+    """Converts a CFG to the Chomsky Normal Form"""
+    return cfg.to_normal_form()
```

Comment:
It seems, that the result grammar is not equal to the source in some cases, note about weak cnf